### ISSUE TITLE
Replace resolve_message_type with roslib.message.get_message_class

### ISSUE THREAD

**Issue by gordonbrander** [OpenAgInitiative/openag_brain#49]
I think we can replace `resolve_message_type` with the `roslib` function `get_message_class`. They both take a message type string and return the Python class for that message type.

``` python
from roslib.message import get_message_class
type_string = "std_msgs/Float32"
MsgType = get_message_class(type_string)
```

**Comment by gordonbrander** [comment]
Here's the source for the roslib function https://github.com/ros/ros/blob/kinetic-devel/core/roslib/src/roslib/message.py#L95

**Comment by LeonChambers** [comment]
Resolved in #74
